Fix doc issues (#2940)
OSInside/kiwi: doc/source/working_with_images/iso_to_usb_stick_grub2_boot_from_iso.rst

the image can be used as a CD/DVD or as a disk. This works because
the ISO image also has a partition table embedded. With more and more
computers delivered without a CD/DVD drive this becomes important.
The deployment of such an image onto a disk like an USB stick normally
destroys all existing data on this device. It is also not possible to use
USB stick as a data storage device. Most USB sticks are pre-formatted

> with a FAT32 or exFAT Windows File System and to keep the existing data

Reviewer: I see some issues with this sentence:

* "Deployment" is a bit heavy for an ISO image. "Writing" is more common for ISOs.
* "Normally" is a dangerous word here. 😉 If you are writing an ISO image to a drive, it **will** overwrite the partition table and data. "Normally" implies there is a chance the data might survive, which gives the user false hope.
* Since you just said the data will be erased, you want to signal that the drive remains unusable for storage *after* the process.

Combining all of them, here is my suggestion:

```suggestion
Writing this image to a USB stick will permanently erase all existing data
on the device. Additionally, the stick will no longer be usable for
general data storage. Most USB sticks are pre-formatted
with a FAT32 or exFAT Windows file system and to keep the existing data
```

on the stick untouched a different deployment needs to be used.

Fortunately Grub2 supports booting directly from ISO files. It does not matter
whether it is installed on your computer's hard drive or on a USB stick.
The following deployment process copies the ISO image as an additional file
to the USB stick or hard drive. The ability to boot from the disk is configured
through a Grub2 feature which allows to loopback mount an ISO file and boot the
kernel and initrd directly from the ISO file.

The initrd loaded in this process must also be able to loopback
mount the ISO file to access the root filesystem and boot the
live system. Almost every Linux distribution supports fat32,
and more and more of them also support exFAT. For hard drives,
Linux filesystems are also supported.

The dracut initrd system used by {kiwi} provides this
feature upstream called as "iso-scan/filename". Therefore all {kiwi} generated
live ISO images supports this deployment mode.

The following procedure expects an existing Grub2 installation on your
hard drive or USB stick for the x86_64 architecture.

1. Copy the ISO image to a folder of your choice on your hard drive.

>    For example, you can create a folder called "iso" in the

Reviewer: Remove filler words:

```suggestion
   For example, create a folder called "iso" in the
```

   root of your hard drive and copy the ISO image there:

   .. code:: bash

      sudo mkdir /iso
      sudo cp some-kiwi-live.iso /iso/

2. Lookup the root filesystem UUID of your current operating system.

>    You can get this information by running the command:

Reviewer: Remove filler words:

```suggestion
   Get this information by running the command:
```

   .. code:: bash

      findmnt -n -o UUID /

> 2. Add the following submenu setup to the `grub.cfg` file:

Reviewer: If I'm not mistaken, you should use the `:file:` macro here:

```suggestion
2. Add the following submenu setup to the :file:`grub.cfg` file:
```

      Be sure to set the correct path to the ISO image, you can set
      your own menu name. The drive identification for Grub2 can be
      checked at boot time by pressing the 'c' key and typing 'ls'.

   .. code:: bash

      menuentry "Boot from Live ISO" {
          load_video
          set gfxpayload=keep
          insmod gzio
          insmod part_gpt
          insmod iso9660
          insmod btrfs
          insmod ext2
          insmod xfs
          insmod lvm
          insmod loopback
          search --no-floppy --fs-uuid --set=root UUID_FROM_STEP_2
          set isofile="/iso/some-kiwi-live.iso"
          set kernel="(loop)/boot/x86_64/loader/linux"
          set initrd="(loop)/boot/x86_64/loader/initrd"
          set options="rd.live.image root=live:CDLABEL=CDROM"
          loopback loop $isofile
          linux $kernel iso-scan/filename=$isofile $options
          initrd $initrd
      }

3. Restart your computer and select the added menuentry.

.. note::

   For USB sticks, the procedure is identical. You would then install
   Grub2 on the USB drive and follow the steps above. However, it is
   uncommon to have Grub2 already installed on a USB stick and there are

>    good helper tools such as `MultiOS-USB` which simplifies the setup.

Reviewer: I would slightly change the order:

```suggestion
   For USB sticks, the procedure is identical. You would install
   Grub2 on the USB drive and follow the steps above. However, it is
   uncommon to have Grub2 already installed on a USB stick. To simplify
   the setup, there are good helper tools such as `MultiOS-USB`.
```
